### PR TITLE
SAK-40427 forums > rename to 'Discussions'

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-40427
https://jira.sakaiproject.org/browse/FARM-2

Consensus was to rename the user facing references from "Forums" to "Discussions".

Conversion script PR here: https://github.com/sakaiproject/sakai-reference/pull/95

### CODE REVIEW

File: library/src/webapp/content/gateway/features.html [hunk 1]
```diff
@@ -231,7 +231,7 @@ <h4 class="modal-title">Communication and Collaboration</h4>
 					<strong>Contact Us</strong> - Quickly and easily direct questions and feedback to the correct source. With one click, the Contact Us tool can send an email to an institution’s helpdesk, with all relevant info automatically included.<br><br>
 					<strong>Email</strong> - Send mail to participants in your site using the institution’s official email system.<br><br>
 					<strong>Email Archive</strong> - Create a mailing list for your course or worksite, allowing for many-to-many broadcasting; access and search an archive of email sent to this address.<br><br>
-					<strong>Forums</strong> - Create, moderate and manage discussion topics and groups within a course and send private messages to participants; options to grade discussions, set word count, permissions, etc.<br><br>
+					<strong>Discussions</strong> - Create, moderate and manage discussion topics and groups within a course and send private messages to participants; options to grade discussions, set word count, permissions, etc.<br><br>
```
Comment (fostersdesign) [inline library/src/webapp/content/gateway/features.html:234]: Please move Discussions up between Contact Us and Email to keep the list alphabetized. Thanks.

**Review by fostersdesign — CHANGES_REQUESTED** [review]
Just one little change, please. Thanks.

File: library/src/webapp/content/gateway/features.html (hunk 1 above)
Comment (bjones86) [reply library/src/webapp/content/gateway/features.html]: Done. Thanks for the review!

File: help/help/src/sakai_screensteps_forumsUserGuide/howdoIdeleteaforumpostieconversation.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+f<!DOCTYPE html>
```
Comment (fostersdesign) [inline help/help/src/sakai_screensteps_forumsUserGuide/howdoIdeleteaforumpostieconversation.html:1]: Rogue "f" here.

**Review by fostersdesign — CHANGES_REQUESTED** [review]
One more fix needed, please.

File: help/help/src/sakai_screensteps_forumsUserGuide/howdoIdeleteaforumpostieconversation.html (hunk 2 above)
Comment (bjones86) [reply help/help/src/sakai_screensteps_forumsUserGuide/howdoIdeleteaforumpostieconversation.html:1]: Damn fingers!

**Review by fostersdesign — APPROVED** [review]
Thanks, @bjones86! Looks good now.

**Pull request merged**